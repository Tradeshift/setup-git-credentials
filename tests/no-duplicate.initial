https://user:password@example.org
