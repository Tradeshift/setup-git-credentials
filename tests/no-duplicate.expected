https://user:password@example.org
https://user:password@example.com
